app/src/main/java/com/swent/assos/ui/screens/News.kt

```diff
@@ -46,12 +55,30 @@ fun News(navigationActions: NavigationActions) {
           if (loading.value) {
             item { LoadingCircle() }
           } else {
-            if (news.isEmpty()) {
-              item { Text(text = stringResource(R.string.NoResult), textAlign = TextAlign.Center) }
-            } else {
-              items(news) { news -> HomeItem(news = news, navigationActions) }
-              item { Spacer(modifier = Modifier.height(10.dp)) }
+            if (followingNews.isNotEmpty())
+                item {
+                  Text(
+                      text = "Activity of associations you follow :",
+                      style = MaterialTheme.typography.headlineMedium,
+                      fontSize = 20.sp,
+                      modifier = Modifier.testTag("followActivityTitle"))
```

Comment:
Add the font we use all over the app and also add the color on background so that it is visible in dark mode